### ISSUE TITLE
A shelving problem ：the nextcloud win10's app adds network disk

### ISSUE THREAD

**Issue by microdockman** [nextcloud/desktop#5065]
I found some discussion about this issue in other forums

I need to add the samba disk that has been mounted locally on win10
![Capture](https://user-images.githubusercontent.com/31898637/196438285-1a4c5d40-b730-4457-9a1e-5e75c3d31649.PNG)

the version that can be successfully added is：nextcloud 3.1.3 (windows) 
installation package: Nextcloud-3.1.3-setup.exe  # .exe abandoned

After that, all versions fail to be added, permission problem. 

Do i need to change something on the samba server ?

**Comment by joshtrichards** [comment]
Please provide specific reproduction steps. It's not entirely clear what you're trying to do.

Also what do you mean - precisely - by "permission problem"?

> After that, all versions fail to be added, permission problem.

**Comment by microdockman** [comment]
> Please provide specific reproduction steps. It's not entirely clear what you're trying to do.
> 
> Also what do you mean - precisely - by "permission problem"?
> 
> > After that, all versions fail to be added, permission problem.

The issue has been resolved. The Samba login user for Windows is set as root, and the Nextcloud Windows client does not have permission to create folders in Samba. I manually created a folder on the server, changed the UID and GID to 33, and the problem was resolved. Subsequently, the contents placed in this folder can be synchronized normally. Thanks for the response.
